### PR TITLE
Login now works for registered users (added sessions). 

### PR DESCRIPTION
**Pull request by apiccato**
Also commented some stuff out of register. Changes login.html, creates two login pages (long/tutor, login/tutee)

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -62,31 +63,32 @@ def register_user(user_type, form):
         account['school'] = form["school"]
         account['grade'] = form["grade"]
         
-        a1 = form["address1"]
```
Comment (apiccato) [inline app.py]: @CobyGoldbergs this is what I commented out

**Comment by CobyGoldbergs** [comment]
Dear friend,

The stuff with zip code, longitude, lat, etc. all came from inputting the address to the googlemaps.py locate function. It shouldn't be commented out, cuz as long as an "address" text box is in register.html, they can all be found. If there is no "address" text box in register, it should be added because we need that for our project

File: app.py [hunk 2]
```diff
@@ -62,31 +63,32 @@ def register_user(user_type, form):
...
-                td = times.split(";")
-                x = 0
-                #each day is given seperate element with value being a dictionary of time, address
-                while x < len(td):
-                        account['%s' % td[x]] = {"time": td[x+1], "address": td[x+2]}
-                        x += 3
-                account['match_score'] = 0 #used in comparing for searches
-        print account
+        ## Commented out below; does not work with current form for register
+        # a1 = form["address1"]
```
Comment (CobyGoldbergs) [inline app.py]: So long as there is that address1, the rest of the stuff with address (zip code, etc.) work fine through googlemaps.py

File: app.py [hunk 3]
```diff
@@ -62,31 +63,32 @@ def register_user(user_type, form):
...
+        # loc = locate(a1) #returns three part array, longtitude, latitude, and zip, for parameter address
+
+        # address1 = {}
+        # address1["longitude"] = loc[0] #longitude
+        # address1["latitude"] = loc[1] #latitude
+        # address1["zipcode"] = loc[2] #zipcode
+        # address1["address"] = a1 #actual address
+        # account["%s_Address" % a1_type] = address1 #store dictionary of all a1's info
+
+        # if user_type == "tutor":
```
Comment (CobyGoldbergs) [inline app.py]: this is old code, works with register_tutor for sure

**Comment by apiccato** [comment]
@CobyGoldbergs got it i uncommented it and everything works!

**Pull request merged**